docs/spec/spec.md

```diff
@@ -651,6 +652,9 @@ The `String` type is represented using the `string` type in Ballerina. It can re
 #### 4.1.4 Boolean
 The `Boolean` type is represented using the `boolean` type in Ballerina.
 
+#### 4.1.5 ID
+The `ID` type is represented using the `@graphql:ID` annotation in Ballerina.
```

Comment:
Shall we add a couple of examples here, since using an annotation is not straightforward?